spec/features/courses/subjects/new_spec.rb

```diff
@@ -155,12 +155,29 @@
       it_behaves_like "a course creation page"
     end
 
+    context "Not selecting master subject" do
+      let(:course) do
+        build(:course, :new, provider: provider, level: level, gcse_subjects_required_using_level: true, edit_options: edit_options)
+         .tap { |course| course.errors.add(:subjects, "Invalid") }
+      end
+      let(:selected_fields) { { subjects_ids: [""] } }
+      let(:build_course_with_selected_value_request) { stub_api_v2_build_course(selected_fields) }
+
+      before do
+        build_course_with_selected_value_request
+      end
+
+      scenario "error flash" do
+        new_subjects_page.continue.click
+        expect(new_subjects_page.error_flash.text).to include("Subjects Invalid")
```

Comment:
Can we have a screenshot please...

It reads wrong when the context is somewhere along the lines of `Please select a subject`

Reply:
<img width="1125" alt="Screenshot 2020-07-08 at 15 49 09" src="https://user-images.githubusercontent.com/50492247/86933822-e2740000-c132-11ea-8909-34a33a3c7da3.png">